apps/dapp/src/constants/env/production.tsx

```diff
@@ -1,16 +1,52 @@
 import { Environment } from './types';
 
 const env: Environment = {
```

Comment:
confirming there are no secrets (and safe to checkin/have in git).

Guessing the dapp doesn't have secrets in general?

Reply:
yeah dapp doesnt have secrets